client/pages/Character.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+
+const Character = ({ characterData, setSubNavBarText }) => {
+  const { name, description, thumbnailUrl, comicAppearances } = characterData;
+  useEffect(() => {
+    setSubNavBarText('CHARACTER');
```

Comment:
I would probably name this `onMount` since you want it to reflect the event that triggered the call. The name `setSubNavBarText` reflects what you want the calling component to do with the event, which puts too much information in this component. It shouldn't care what its parent does.

In fact, you could just call `onMount()` and let the parent determine that when Character mounts that means it should set the subnavbar text to 'CHARACTER'.